networking/ingress/getting-traffic-cluster.adoc

```diff
@@ -0,0 +1,101 @@
+[id="getting-traffic-cluster"]
+= Getting traffic into a cluster
+include::modules/common-attributes.adoc[]
+:context: getting-traffic-cluster
+
+toc::[]
+
+{product-title} provides multiple methods for communicating from
+outside the cluster with services running in the cluster.
+
+[NOTE]
+====
+The procedures in this section require prerequisites performed by the cluster
+administrator.
+====
+
+Administrators can expose a service endpoint that external traffic can reach, by
+assigning a unique external IP address to that service from a range of external
+IP addresses. Administrators can designate a range of addresses using a CIDR
+notation, which allows an application user to make a request against the cluster
+for an external IP address.
+
+Each IP address should be assigned to only one service to ensure that each
+service has a unique endpoint. Potential port clashes are handled on a
+first-come, first-served basis.
+
+The recommendation, in order or preference, is:
```

Comment:
The assembly starts by saying there are multiple methods, and then it describes one method (external IPs), but it has not described the other methods yet.  It would flow better to introduce all methods before recommending how to choose which one to use.

Reply:
I commented this out for now until we can address it further.

Reply:
I think it would be clearer if we kept the first paragraph ("{product-title} provides methods for communicating from outside the cluster with services running in the cluster.") and commented out the paragraphs that are specifically about external IPs (the two paragraphs starting with, "Administrators can expose a service endpoint that external traffic can reach").
assigning a unique external IP address" through "